routes/renderers.js

```diff
@@ -85,12 +85,12 @@ function _renderDashboard(req, res, next, opts) {
         .then(function success(notebook) {
             debug('Success loading nb');
 
-            // Check if some cell in the notebook has the dashboard metadata
-            var hasDashboardLayout = notebook.cells.some(function(currentCell){
-                return ( currentCell.hasOwnProperty('metadata') &&
-                         currentCell.metadata.hasOwnProperty('urth') &&
-                         currentCell.metadata.urth.hasOwnProperty('dashboard') );
-            });
+            // get dashboard layout from notebook-level dashboard metadata
+            var dashboardLayout = (notebook.metadata &&
+                                  notebook.metadata.urth &&
+                                  notebook.metadata.urth.dashboard &&
+                                  notebook.metadata.urth.dashboard.layout) ||
+                                  'no-layout';
```

Comment:
We decided no layout metadata meant render as a report here. But in the extension we said if layout was missing, we'd look for grid metadata and assume grid is the default. Hmmm.

If I take a notebook that was produced by the 0.4.x extension and drop it in dashboard server now with this change, it's going to render in report layout and ignore all the grid metadata. That seems ... unfriendly.

Can we at least do a sniff test to see if any of the cells have the `urth.dashboard` path. If any of them do but layout is undefined in the notebook level metadata, default to grid. If there's no urth.dashboard anywhere then take the no-layout route.